### PR TITLE
Create SeiTime;Amalgatrix.json

### PR DESCRIPTION
**Pull request by DMSei**
Amalgatrix Species was created and converted by me, SeiTime.

### CODE REVIEW

File: race/SeiTime;Amalgatrix.json [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+				"abbreviation": "AMIX",
+				"json": "AMIX Lineage",
+				"full": "Amalgatrix",
+				"authors": "Sei Time",
+				"url": "https://www.dndbeyond.com/races/1136727-amalgatrix",
+				"version": "2023-03-16"
+			}
+		]
+	},
+	"name": "Amalgatrix",
```
Comment (hakr14) [inline race/SeiTime;Amalgatrix.json:14]: This and the following properties should be within an object in the `race` array. So it should look like:
```json
"race": [
    {
        "name": "Amalgamatrix",
        "source": "AMIX",
        ...
```

File: race/SeiTime;Amalgatrix.json [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+			"type": "entries",
+			"name": "Languages",
+			"entries": [
+				"You can speak, read, and write Common and one other language of your choice. Amalgatrix live in many locations and could have adapted any number of different varieties of languages."
+			]
+		}
+	]
+	}
+	],
+	"fluff": {
```
Comment (hakr14) [inline race/SeiTime;Amalgatrix.json:134]: The "fluff" object should be included in the above "race" object.

**Review by hakr14 — CHANGES_REQUESTED** [review]

File: race/SeiTime;Amalgatrix.json [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+			"items": [
+				"{@b Dwarves.} “Dwarves have no regard for their natural surroundings, destroying forests and mountains just to get at the deeper buried minerals. It is all that seems to matter to them, no matter how many times we try to teach them.”",
+				"{@b Halflings.} “Halflings are simple people who enjoy simple pleasures. They often scurry about the more dangerous races with abandon. They do not tend to destroy nature around them, so they are ideal traveling companions.”",
+				"{@b Humans.} “Humans are a worse version of Dwarves, at least Dwarves tend to stay below nature... Humans warp and twist nature to suit their needs; like a plague of locusts, they just keep expanding. At least they do not taste terrible.”",
+				"{@b Elves.} “Elves are among the nicest of the races. They care for their environment and even give back to it as much as they can. Of all the bipedal races, they can be treated with great respect.”"
+					]
+				}
+			]
+		},
+		[
```
Comment (hakr14) [inline race/SeiTime;Amalgatrix.json:204]: Delete

File: race/SeiTime;Amalgatrix.json [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+			"{@b Pup Names:} Ithil, Orn, Anor, Aew, Aduial, Alf, Celu, Ethuil, Celeth, Loth, Meril, Eirien, Glawar, Nen, Ross, Eilia",
+			"{@b Adult Names:} Andreth, Adleitha, Amdir, Anira, Baneth, Braig, Bronwe, Celeg, Craim, Cyron, Daedelu, Doron, Draug, Duinen, Egleria, Eilian, Espalass, Faen,Fanya, Ferais, Gaear, Galadh, Gell(a), Guenivhar, Gwaedh, Gwilwileth, Hal, Hartha, Hithu, Idhor, Imloth, Ivren, Laer, Lalaith, Lista, Mele, Mavras, Minuial, Narn, Naur, Nim, Ninniach, Oll, Orod, Parth, Pessa, Puig, Randir, Rhaw, Ryn, Sador, Sidh, Suil, Taig, Teilia, Thelyn, Uial, Ungol, Uthaes",
+			"{@b Pack Names (Common Translations):} Airavanessë (Eternalbeauty), Moralta (Nightprosper), Calaliltië(Lightdancing), málosnestando(Foresthealer), Toltiris(Islandguard), Airenlomba(Seasecret), Vilyatirmo(Skywatcher), Rottofarasta(Tunnelhunter), Ostotirmo(Citywatcher), Farasaicë(Hunthunger), Ennananwa(Firstexisting)"
+					]
+				}
+			]
+		}
+	]
```
Comment (hakr14) [inline race/SeiTime;Amalgatrix.json:223]: Delete

File: race/SeiTime;Amalgatrix.json [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+},
+			{
```
Comment (hakr14) [inline race/SeiTime;Amalgatrix.json:225]: Delete

File: race/SeiTime;Amalgatrix.json [hunk 6]
```diff
@@ -0,0 +1,256 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"abbreviation": "AMIX",
+				"json": "AMIX Lineage",
+				"full": "Amalgatrix",
+				"authors": "Sei Time",
```
[the remark this answers is not in the document]
Comment (hakr14) [reply race/SeiTime;Amalgatrix.json:8]: So this line is `"authors": ["Sei Time"],`

**Review by hakr14 — CHANGES_REQUESTED** [review]

**Comment by hakr14** [comment]
In future you really should be using an IDE that can validate JSON, such as VSCode.

**Comment by DMSei** [comment]
> In future you really should be using an IDE that can validate JSON, such as VSCode.

Every edit I've made and the original upload I have done in Notepad++, and then verified in Virtual Studio Code before submission. Only the second last edit/upload had errors in VSCode. This is my first time creating a race json. There is bound to be issues, or errors that need correcting. I apologize for my lack of experience in this matter.

**Comment by DMSei** [comment]
I don't have any errors in VSCode anymore. Is this good and properly formatted for use now?

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]
Not quite; peek the "Checks" tab at the top of the PR: [https://github.com/TheGiddyLimit/homebrew/pull/1670/checks](https://github.com/TheGiddyLimit/homebrew/pull/1670/checks)

**Comment by DMSei** [comment]
@TheGiddyLimit Thanks. I made some alterations to the file in order to get past the error I was experiencing. It is still saying there is an error when the check runs saying that AMIX is not in the _meta source. But it is. Do you know what it is I am doing wrong?

**Review by TheGiddyLimit — APPROVED** [review]

**Pull request merged**